data/registry/Dockerfile

```diff
@@ -6,6 +6,8 @@ ADD go-docker-registry.deb /tmp/go-docker-registry.deb
 ADD config.json /etc/go-docker-registry/config.json
 RUN dpkg -i /tmp/go-docker-registry.deb || true
 
+
 # Ew... the deb waits for the manager to start up, which requires runsv.
 RUN runsvdir /etc/service & svpid=$! && apt-get install -f -y && kill $svpid
 #RUN apt-get install -f -y || true
+
```

Comment:
I guess I changed the file for some experiment only to find out it was useless; but somehow get the file checked in instead of revert; any way, restored to its original state.